spec/models/dialog_field_sorted_item_spec.rb

```diff
@@ -104,14 +105,14 @@
         context "when the default_value is not included in the list of values" do
           let(:default_value) { "test4" }
 
-          it "uses the first value as the default" do
+          it "uses the given value as the default" do
             dialog_field.initialize_value_context
-            expect(dialog_field.default_value).to eq("test1")
+            expect(dialog_field.default_value).to eq("test4")
           end
 
           it "sets the values to what should be returned from automate" do
```

Comment:
This description should probably be changed since it's not actually returning values from automate anymore.

Reply:
Thanks, fixed :)